types/js2coffee/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "source-map": "^0.7.3"
```

Comment:
This should use different minor version:
https://github.com/js2coffee/js2coffee/blob/47d3159aba94ffd4377138f7837717d8628eef9f/package.json#L37
But, I think this is ok to use that specific version published `d.ts` types, intead of latest @types/source-map
Am I correct?

Reply:
We could go with this version: https://github.com/mozilla/source-map/blob/0.5.7/source-map.d.ts

Reply:
Oh, ok. So please review this topic. I'd say, safer is to use @types/source-map@.... as dependency in `package.json` (I understand the source for package is out of DT already, so you need entry in `package.json`, I understand this @types/source-map could require whitelisting, if you decide to use @types/source-map)
I can see now why you've added native packages dependency to `packages.json`, as the original types sources are no longer here

Reply:
The removal process was wrong, I think you have to go with 0.7.7 version (sorry for the hassle).
The passghrouh points to distribution which has incorrect ntry in NPM without those types shipped, so
the DT ill never pass a CI (nor install proper deps):
```
npm notice package: source-map@0.5.7
npm notice === Tarball Contents ===
npm notice 2.6kB   package.json
npm notice 23.5kB  README.md
npm notice 1.5kB   LICENSE
npm notice 405B    source-map.js
npm notice 7.9kB   CHANGELOG.md
npm notice 260.2kB dist/source-map.debug.js
npm notice 101.9kB dist/source-map.js
npm notice 26.3kB  dist/source-map.min.js
npm notice 245.8kB dist/source-map.min.js.map
npm notice 3.2kB   lib/array-set.js
npm notice 4.7kB   lib/base64-vlq.js
npm notice 1.5kB   lib/base64.js
npm notice 4.2kB   lib/binary-search.js
npm notice 2.3kB   lib/mapping-list.js
npm notice 3.6kB   lib/quick-sort.js
npm notice 38.4kB  lib/source-map-consumer.js
npm notice 14.1kB  lib/source-map-generator.js
npm notice 13.8kB  lib/source-node.js
npm notice 10.5kB  lib/util.js
npm notice === Tarball Details ===
```